Modules/System/Cryptography Management/src/Rfc2898DeriveBytes.Codeunit.al

```diff
@@ -0,0 +1,31 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+codeunit 50101 "Rfc2898DeriveBytes"
```

Comment:
The codeunit itself needs to be documented like in https://github.com/microsoft/ALAppExtensions/blob/c07cb0dc989fdd022a9dc18eb8db22a190b5fefb/Modules/System/Cryptography%20Management/src/RijndaelCryptography.Codeunit.al#L6-L8

Reply:
done!